packages/@o3r/eslint-plugin/src/rules/json/json-dependency-versions-harmonize/json-dependency-versions-harmonize.spec.ts

```diff
@@ -113,23 +95,41 @@ ruleTester.run('json-dependency-versions-harmonize', jsonDependencyVersionsHarmo
             depName: 'myDep',
             packageJsonFile: path.join(fakeFolder, 'local', 'package.json'),
             version: '^2.0.0'
-          }
+          },
+          suggestions: [
+            {
+              messageId: 'versionUpdate',
+              data: {
+                version: '^2.0.0'
+              },
+              output: JSON.stringify({ dependencies: { myDep: '^2.0.0' } })
+            }
+          ]
         }
       ]
     },
     {
       filename: packageToLint,
       output: JSON.stringify({ peerDependencies: { myOtherDep: '~2.0.0' } }),
       code: JSON.stringify({ peerDependencies: { myOtherDep: '^2.0.0' } }),
-      options: [{ alignPeerDependencies: true }],
+      options: [{ alignPeerDependencies: true } as VersionsHarmonizeOptions],
```

Comment:
Needed else the following tests expect to have the same options type, I don't understand why because typings seems okay...